Doist/todoist-google-sheets
# chore: Update Google consent screenshot

## Overview

Update Google consent screenshot to have a checked box. Also, update copy text to mention only one box. 

## Reference

- Follow up to https://github.com/Doist/todoist-google-sheets/pull/52

## Test Plan

- Create Google Sheet extensions (context-menu and settings) in todoist.com/app_console (more details in previous PRs)
- Trigger the extension
- Do not select the last box in the Google consent screen
<img width="498" alt="Screenshot 2022-09-14 at 10 54 55" src="https://user-images.githubusercontent.com/1509326/190111095-68655844-ba7f-4764-936c-9487b35d3915.png">

- Check that you get a warning telling you you should check the last box in the consent screen
<img width="600" alt="Screenshot 2022-09-14 at 11 00 33" src="https://user-images.githubusercontent.com/1509326/190111207-37681c17-b2af-4d9f-9b82-5a4da689e937.png">


## Creator Checklist


## Reviewer Checklist
